spec/features/admin/budgets_spec.rb

```diff
@@ -170,5 +170,27 @@
       end
     end
 
+    scenario 'Calculate Heading winners', :js, :winners do
+      budget = create(:budget, name: 'Yearly participatory budget')
+      group  = create(:budget_group, budget: budget, name: 'Districts improvments')
+      heading  = create(:budget_heading, group: group, price: 5)
+      unselected_investment = create(:budget_investment, :unselected, heading: heading, ballot_lines_count: 3, price: 4)
```

Comment:
Winner selection is simple: Take all selected investments, in ballot_lines_count order, until we run out of price :D (classic [Knapsack problem](https://en.wikipedia.org/wiki/Knapsack_problem))